static/css/themes/board.css

```diff
@@ -387,7 +387,9 @@ textarea.expanded {
 
 .card-actions button i[class^="icon-"],
 .card-actions .btn i[class^="icon-"] {
-    margin-right: 3px;
+    margin-right: 5px;
+    position: relative;
+    top: 1px;
```

Comment:
As icons are a little bit bigger, they are not properly align with the nearby text, so I had to move them a pixel away.